src/bindings/python/src/openvino/runtime/ie_api.py

```diff
@@ -65,18 +64,18 @@ def get_input_types(obj: Union[InferRequestBase, CompiledModelBase]) -> dict:
     def get_inputs(obj: Union[InferRequestBase, CompiledModelBase]) -> list:
         return obj.model_inputs if isinstance(obj, InferRequestBase) else obj.inputs
 
-    def map_tensor_names_to_types(input: ConstOutput) -> dict:
-        return {n: input.get_element_type() for n in input.get_names()}
+    def map_tensor_names_to_types(inpt: ConstOutput) -> dict:
+        return {n: inpt.get_element_type() for n in inpt.get_names()}
 
     input_types: dict = {}
-    for idx, input in enumerate(get_inputs(obj)):
+    for idx, inpt in enumerate(get_inputs(obj)):
```

Comment:
may be better to rename it to `input_value`

Reply:
changed